File: backend/Multibanca.Domain.Models/Multibanca/revisar_liquidacion.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class revisar_liquidacion : base_auditoria
    {
        public int id_revisar_liquidacion { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public bool is_enviar_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/revisar_liquidacion/models/revisar_liquidacion.ts
export interface RevisarLiquidacion {
  id_revisar_liquidacion: number;
  id_expediente: number;
  id_usuario_solicitante: number;
  is_enviar_reparo: boolean;
  observaciones?: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
